src/vm/types/range_check_instance_def.zig

```diff
@@ -3,10 +3,29 @@ pub const CELLS_PER_RANGE_CHECK: u32 = 1;
 
 /// Represents a Range Check Instance Definition.
 pub const RangeCheckInstanceDef = struct {
+    const Self = @This();
+
     /// Ratio
     ratio: ?u32,
     /// Number of 16-bit range checks that will be used for each instance of the builtin.
     ///
     /// For example, n_parts=8 defines the range [0, 2^128).
     n_parts: u32,
+
+    /// Number of cells per built in
+    pub fn cellsPerBuiltin() u32 {
```

Comment:
Why do we need a function here as the `CELLS_PER_RANGE_CHECK` constant is declared as public as so already available? If it is still necessary to have this function, shouldn't we declare a constant in the structure which directly consumes this function at compile time to speed up the calculations? 

You have examples of that [here](https://kristoff.it/blog/what-is-zig-comptime/)

![image](https://github.com/keep-starknet-strange/cairo-zig/assets/60488569/2672985c-0642-48cc-af4f-7aee57b6f542)